yt/visualization/plot_window.py

```diff
@@ -1103,12 +1104,30 @@ def setup_callbacks(self):
             callback.__doc__ = CallbackMaker.__doc__
             self.__dict__['annotate_'+cbname] = types.MethodType(callback,self)
 
-    @invalidate_plot
     def annotate_clear(self, index=None):
         """
         Clear callbacks from the plot.  If index is not set, clear all
         callbacks.  If index is set, clear that index (ie 0 is the first one
         created, 1 is the 2nd one created, -1 is the last one created, etc.)
+
+        .. note::
+
+            Deprecated in favor of `clear_annotations`.
+
```

Comment:
I think the deprecation notice should be at the start of this docstring (it doesn't need to be in a note). If a user looks at this function, the very first thing they read should be that it has been deprecated in favor of clear_annotations.